puppet/instrumental_tools/templates/instrumental.yml.erb

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Yeah, absolutely agreed. I could be swayed on changing this now, but this PR is so weighty already that yeah, I'd prefer to wait for next cycle.